llvm/lib/Target/M68k/M68kISelLowering.cpp

```diff
@@ -51,7 +51,7 @@ M68kTargetLowering::M68kTargetLowering(const M68kTargetMachine &TM,
 
   MVT PtrVT = MVT::i32;
 
-  setBooleanContents(ZeroOrOneBooleanContent);
+  setBooleanContents(ZeroOrNegativeOneBooleanContent);
```

Comment:
could you add a brief comment here to note that we're using SETCC's boolean interpretation.